ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5655,4 +5655,120 @@ RFVSC2X:
   - return_fan_run_status_2
   - return_fan_speed_percentage_command_2
   implements:
-  - OPERATIONAL
\ No newline at end of file
+  - OPERATIONAL
+
+EXTOR:
+  description: "shade control with extension monitoring based on outside illuminance"
+  is_abstract: true
+  uses:
+  - shade_extent_percentage_command
+  - outside_illuminance_sensor
+  - low_limit_illuminance_setpoint
+  - high_limit_illuminance_setpoint
+
+DPHC2:
+  description: "Two-pipe heating and cooling control for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - supply_water_temperature_setpoint
+  uses:
+  - supply_water_temperature_sensor
+  - return_water_temperature_sensor
+  - water_riser_mode
+  - cooling_output_percentage_command
+  - heating_output_percentage_command
+
+DPHC4:
+  description: "Four pipe heating and cooling controls for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - heating_supply_water_temperature_setpoint
+  - chilled_supply_water_temperature_setpoint
+  uses:
+  - heating_supply_water_temperature_sensor
+  - heating_return_water_temperature_sensor
+  - chilled_supply_water_temperature_sensor
+  - chilled_return_water_temperature_sensor
+  - cooling_output_percentage_command
+  - heating_output_percentage_command
+
+HHCD42:
```

Comment:
Since this type is so strange, can you provide a diagram of this? You can email me directly. I'd like to write a doc for the repo that specifies how we model these when we see them, using this as a template.

Reply:
I did this because we are using 6-way valves for the RP's on 4-pipe installations. In this case the command of the value represents either 0-100% cooling (e.g. 2-4.7 V) or heating (e.g. 5.3 to 10V, see picture).  Therefore I thought it would be of use to represent the cooling or heating load seperately, but these values do not represent the the valve position...
![2022-12-01 11_24_41-belimodatasheet](https://user-images.githubusercontent.com/82176022/205031112-b66698a0-cde7-484b-b1cf-f89de5b82769.png)

Reply:
Any proposal for a call?